src/porepy/utils/error.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import annotations
```

Comment:
While we are at it, is `error.py` a good name for this module? I am tempted to suggest `error_computation` or similar. What do you think?

Reply:
Yes, totally agree. `error_computation` is way more descriptive. I've renamed the module.